FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -3115,14 +3133,24 @@ subroutine GFS_physics_driver                                 &
 !
 
       do k=1,levs
-        do i=1,im
-          Stateout%gt0(i,k)  = Statein%tgrs(i,k) + dtdt(i,k) * dtp
-          Stateout%gu0(i,k)  = Statein%ugrs(i,k) + dudt(i,k) * dtp
-          Stateout%gv0(i,k)  = Statein%vgrs(i,k) + dvdt(i,k) * dtp
+         do i=1,im
+#ifdef SUBSET_PHYSICS
+        Stateout%gt0(i,k)  = Statein%tgrs(i,k) ! + dtdt(i,k) * dtp [EW]: override this since we can't turn off moninq
```

Comment:
Maybe delete the comment? I'm not sure posterity will know who EW is. They can look at the git log to figure out who wrote the commit that introduced the comment.

Reply:
good point, I'll change it, thank you.